docs/sources/setup/kubernetes.md

```diff
@@ -266,14 +267,14 @@ Because of the AppArmour restriction, to run Beyla as unprivileged container, yo
 - Set `container.apparmor.security.beta.kubernetes.io/beyla: "unconfined"` in your Kubernetes deployment files.
 - Set a modified AppArmour policy which allows Beyla to perform `mount`.
 
-**Note** Since the `beyla` container does not have the privileges required to mount or un-mount the BPF filesystem, this sample leaves the BPF filesystem mounted on the host, even after the sample is deleted. This samples uses a unique path for each namespace to ensure re-use the same mount if Beyla is re-deployed, but to avoid collisions if multiple instances of Beyla is run in different namespaces. 
+**Note** Since the `beyla` container does not have the privileges required to mount or un-mount the BPF filesystem, this sample leaves the BPF filesystem mounted on the host, even after the sample is deleted. This samples uses a unique path for each namespace to ensure re-use the same mount if Beyla is re-deployed, but to avoid collisions if multiple instances of Beyla is run in different namespaces.
 
-**Note** Loading BPF programs requires that Beyla is able to read the Linux performance events, or at least be able to execute the Linux Kernel API `perf_event_open()`. 
+**Note** Loading BPF programs requires that Beyla is able to read the Linux performance events, or at least be able to execute the Linux Kernel API `perf_event_open()`.
```

Comment:
In addition to this, it may be worth at least mentioning https://docs.kernel.org/admin-guide/sysctl/kernel.html#unprivileged-bpf-disabled

Reply:
Thank you @rafaelroquetto I'll do another pass to update further content. This pass focused on improving the information architecture and deprecated articles. See **Next steps** in the [description](https://github.com/grafana/beyla/pull/1259#issue-2588439168).